### PR TITLE
Small modifications for documentation

### PR DESCRIPTION
**Pull request by brfrn169**
Please take a look!

### CODE REVIEW

File: docs/multi-storage-transactions.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ scalar.db.multi_storage.namespace_mapping=user:cassandra,coordinator:mysql
 scalar.db.multi_storage.default_storage=cassandra
 ```
 
-## References
+## Further documentation
```
Comment (feeblefakie) [inline docs/multi-storage-transactions.md:52]:
```suggestion
## Further documents
```
Probably. I think Documentation sounds more like creating a document.

File: docs/scalardb-server.md [hunk 2]
```diff
@@ -141,4 +141,6 @@ scalar.db.grpc.deadline_duration_millis=60000
 
 ## Further documentation
```
Comment (feeblefakie) [inline docs/scalardb-server.md:142]: Ditto.

**Review by feeblefakie — COMMENTED** [review]
Left minor suggestions, but other than that, LGTM! Thank you!

**Review by feeblefakie — DISMISSED** [review]

**Review by feeblefakie — DISMISSED** [review]
LGMT! Thank you!

File: docs/multi-storage-transactions.md (hunk 1 above)
Comment (brfrn169) [reply docs/multi-storage-transactions.md]: I fixed it in b80bcbd9bae2806b55c39106ac41cd08ac9a59e0. I also modified the same for other parts.

**Comment by feeblefakie** [comment]
@brfrn169 
A bit picky, but "Further reading" or "Related documents" might be more common.

**Comment by brfrn169** [comment]
> @brfrn169 A bit picky, but "Further reading" or "Related documents" might be more common.

Sure. I will replace it with "Further reading". Thank you!

**Comment by brfrn169** [comment]
@Torch3333 Merging this for now. Please take a look later when you gat a chance!

**Pull request merged**